server/src/main/java/io/druid/server/initialization/BatchDataSegmentAnnouncerConfig.java

```diff
@@ -37,6 +37,18 @@
   @Min(1024)
   private long maxBytesPerNode = 512 * 1024;
 
+  // Skip LoadSpec from segment announcements
+  @JsonProperty
+  private boolean skipLoadSpec = false;
+
+  // Skip dimension list from segment announcements
+  @JsonProperty
+  private boolean skipDimensions = false;
```

Comment:
do we need separate configs for dimensions and metrics? Is there any use in having one but not the other?